views/index.html

```diff
@@ -136,6 +136,7 @@ <h2 class="call-to-action">Let's Talk!</h2>
       <div class="contact-info">
         <span class="contact-detail">Tel: <a href="tel:07708796446">07708796446</a></span>
         <span class="contact-detail">Email: <a href="mailto:hello@dwyl.io">hello@dwyl.io</a></span>
+        <span class="contact-detail">Address: <a href="https://www.google.co.uk/maps/place/16+Palmers+Rd,+London+E2+0SY/@51.5294086,-0.0444392,17z/data=!4m13!1m7!3m6!1s0x48761d2695efb1b1:0x30455a5f7ce8535d!2s16+Palmers+Rd,+London+E2+0SY!3b1!8m2!3d51.5294086!4d-0.0422505!3m4!1s0x48761d2695efb1b1:0x30455a5f7ce8535d!8m2!3d51.5294086!4d-0.0422505">16 Palmers Road, London E2 0SY</a></span>
```

Comment:
this link contains user-specific metadata. https://www.google.co.uk/maps/place/16+Palmers+Rd,+London+E2+0SY is enough. 
also, did you see: https://github.com/dwyl/dwyl-site/pull/205#discussion_r101555817